modules/course_management/course_activator.py

```diff
@@ -28,14 +32,16 @@ async def deactivate_all_courses(interaction: nextcord.Interaction):
 		await deactivate_course(ctx, channel)
 
 
-async def __move_course(interaction: nextcord.Interaction, channel: nextcord.TextChannel, active: bool):
+async def __move_course(
+	interaction: nextcord.Interaction, channel: nextcord.TextChannel, active: bool
+):
 	source_label = INACTIVE_COURSES_CATEGORY if active else ACTIVE_COURSES_CATEGORY
 	target_label = ACTIVE_COURSES_CATEGORY if active else INACTIVE_COURSES_CATEGORY
 	if not channel.category_id == get_id(source_label):
 		raise FriendlyError(
 			f"You can only {'activate' if active else 'deactivate'} a course which is"
 			f" in the {'inactive' if active else 'active'} courses category",
-			ctx,
+			ctx.send,
 			ctx.author,
```

Comment:
ctx was changed to interaction

Reply:
Yep, just put that there so I don't forget when we get to that module